src/main/java/baubles/common/items/ItemRing.java

```diff
@@ -18,33 +14,48 @@
 import net.minecraft.util.EnumHand;
 import net.minecraft.util.NonNullList;
 import net.minecraft.world.World;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
+import baubles.api.BaubleType;
+import baubles.api.BaublesApi;
+import baubles.api.IBauble;
+import baubles.api.cap.IBaublesItemHandler;
 
-public class ItemRing  extends Item implements IBauble
+@Mod.EventBusSubscriber
+public class ItemRing extends Item implements IBauble
 {
+	public static final Item RING = (new ItemRing()).setUnlocalizedName("Ring").setRegistryName("ring");
```

Comment:
Items and blocks should be created in the corresponding registry events. If we need access to this item instance, we should use the `@ObjectHolder` annotation in order to support item substitution properly. Otherwise if a mod overrides this ring, weired bugs or even crashes might occur (since this instance is no longer the one that is registered to the item registry)